src/main/scala/com/gd/twitterstreamingtToDatalake/tweetsIngestion.scala

```diff
@@ -18,7 +18,7 @@ object TweetsIngestion {
 
   def getTweets = {
 
-    val filters: Seq[String] = Seq("uscis")
+    val filters: Seq[String] = Seq("narendramodi")
```

Comment:
could you please add a comment explaining why the particular filter is used? I suppose the filter will influence the target for analysis / purpose and meaning of final reports.

Reply:
These are for filtered tweets only for the required topics.
Have updated the code to remove it as hardcode and provide in configuration if required.